frontend/src/modules/Folders/components/FolderOverview.js

```diff
@@ -28,7 +28,7 @@ export const FolderOverview = (props) => {
       <Grid item lg={4} xl={3} xs={12}>
         <ObjectMetadata
           environment={folder.dataset.environment}
-          region={folder.dataset.region}
+          region={folder.restricted?.region || 'UNAUTHORIZED_INFO'}
```

Comment:
* This is already handled in [ObjectMetadata](https://github.com/data-dot-all/dataall/blob/eca354e8df8189c2fcf31732579def69bad90bfb/frontend/src/design/components/ObjectMetadata.js#L127).
* I think it's better not to print region at all than printing "Region -> UNAUTHORIZED"

ditto for the rest of your special handling

Reply:
In the latest commit 4b675f3 most views have been simplified except for the tables and folder view where table names are presented in the rows.